include/EVT/dev/buttonLEDs.hpp

```diff
@@ -0,0 +1,34 @@
+#ifndef _EVT_buttonLEDs_
+#define _EVT_buttonLEDs_
+
+#include <EVT/dev/LED.hpp>
+#include <EVT/dev/button.hpp>
+#include <EVT/io/GPIO.hpp>
+#include <stdint.h>
+
+namespace EVT::core::DEV {
+
+class ButtonLEDs : Button {
```

Comment:
Do we have the ButtonLEDs implemented? If not, we probably shouldn't have the header for it

Reply:
ButtonLEDS is most likely unnecessary and will be deleted since there is already an LED module in EVT-Core